sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/implementation/SparkBridgeImplementationInternal.scala

```diff
@@ -361,14 +361,14 @@ private[cosmos] object SparkBridgeImplementationInternal extends BasicLoggingTra
   }
 
   def overrideDefaultTcpOptionsForSparkUsage(): Unit = {
-    val overrideJson = "{\"timeoutDetectionEnabled\": true, \"timeoutDetectionDisableCPUThreshold\": 70.0," +
-      "\"timeoutDetectionTimeLimit\": \"PT600S\", \"timeoutDetectionHighFrequencyThreshold\": 100," +
+    val overrideJson = "{\"timeoutDetectionEnabled\": true, \"timeoutDetectionDisableCPUThreshold\": 75.0," +
```

Comment:
are these change intentional?

Reply:
yes - want your eyes on them especially. Background - there is sometimes a straggling single worker - form the errors because it has transit timeouts to single partition - backend is fine (other executors succeeed) - so these changes try to make a better compromise between not being too noisy with bad connection detection but still be able to at least identify bad connection in 1-2 minutes